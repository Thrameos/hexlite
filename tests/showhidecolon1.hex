a(0..2,0..2).

{ b(X,Y) } :- a(X,Y).

c(X) :- b(X,X).

#show b(0,0..2).
#show c/1.
