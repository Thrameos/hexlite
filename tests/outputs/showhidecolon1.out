{b(0,1),b(0,2)}
{b(0,1),b(0,2)}
{c(1),b(0,2),b(0,1)}
{c(1),b(0,2),b(0,1)}
{b(0,1),b(0,2)}
{c(1),b(0,2),b(0,1)}
{b(0,1),b(0,2)}
{c(1),b(0,2),b(0,1)}
